broker/src/main/java/io/camunda/zeebe/broker/transport/commandapi/CommandApiRequestReader.java

```diff
@@ -103,6 +105,14 @@ public void wrap(final DirectBuffer buffer, final int offset, final int length)
       value = recordSupplier.get();
       value.wrap(buffer, valueOffset, valueLength);
     }
+
+    commandRequestDecoder.skipValue();
+    if (commandRequestDecoder.limit() != buffer.capacity()) {
```

Comment:
:thinking: This will be problematic if in the future we add a new variable-length property to the `ExecuteCommandRequest` since we won't be able to distinguish between if the content is from the `authorization` property or from the new one.

Wouldn't how we solved it for the [`RecordMetadata`](https://github.com/camunda/zeebe/blob/main/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/RecordMetadata.java#L108-L115) work here as well?